docs/tutorials/hospital_admissions_model.qmd

```diff
@@ -296,9 +340,7 @@ hosp_model.run(
 )
 ```
 
-We can use `arviz` to visualize the model fit. Here, we plot the observed values against the inferred latent values (i.e. the mean of the negative binomial observation process)[^capture]:
-
-[^capture]: The output is captured to avoid `quarto` from displaying the output twice.
+We can use `arviz` to visualize the model fit. Here, we plot the observed values against the inferred latent values (i.e. the mean of the negative binomial observation process).
```

Comment:
Link to ArviZ here, rather than in the following `Results exploration and MCMC diagnostics` section. Standardize on ArviZ or `arviz`.